src/styles/utils/_variables.scss

```
@@ -0,0 +1,15 @@
+$mainTextColor: #060b35;
```

```suggestion
$main-text-color: #060b35;
```
it's better to name variables with kebab-case